src/components/customers/CustomerSettings.tsx

```diff
@@ -0,0 +1,149 @@
+import { useRef } from 'react'
+import { gql } from '@apollo/client'
+import styled from 'styled-components'
+
+import { SideSection } from '~/styles/customer'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { Button, Typography } from '~/components/designSystem'
+import { theme, NAV_HEIGHT } from '~/styles'
+import {
+  EditCustomerVatRateFragmentDoc,
+  CustomerVatRateFragment,
+  CustomerInvoiceGracePeriodFragment,
+} from '~/generated/graphql'
+import { VAT_RATE_ROUTE } from '~/core/router'
+import { intlFormatNumber } from '~/core/intlFormatNumber'
+import {
+  EditCustomerVatRateDialog,
+  EditCustomerVatRateDialogRef,
+} from '~/components/customers/EditCustomerVatRateDialog'
+
+import {
+  EditCustomerInvoiceGracePeriodDialog,
+  EditCustomerInvoiceGracePeriodDialogRef,
+} from './EditCustomerInvoiceGracePeriodDialog'
+
+gql`
+  fragment VatRateOrganization on Organization {
+    id
+    vatRate
+  }
+
+  fragment CustomerVatRate on CustomerDetails {
+    id
+    vatRate
+    ...EditCustomerVatRate
+  }
+
+  fragment CustomerInvoiceGracePeriod on CustomerDetails {
+    id
+    invoiceGracePeriod
+  }
+
+  ${EditCustomerVatRateFragmentDoc}
+`
+
+interface CustomerSettingsProps {
+  customer: CustomerVatRateFragment & CustomerInvoiceGracePeriodFragment
+}
+
+export const CustomerSettings = ({ customer }: CustomerSettingsProps) => {
+  const { translate } = useInternationalization()
+  // const { currentOrganization } = useCurrentUserInfosVar()
```

Comment:
Peut-être tej ça ?

Reply:
yep ! good catch

Reply:
Will remove in a following PR, if you don't mind :)